### PR TITLE
post: Custom components Dashboard

### PR DESCRIPTION
**Pull request by ZJvandeWeg**

### CODE REVIEW

File: src/blog/2023/10/custom-vuetify-components-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## Custom components
+
+While going through the list of components on [Vuetify](https://vuetifyjs.com/en/components/)
+there's several examples that aren't natively implemented in Dashboard 2.0.
+One example we'll use in a dashboard in this post is the
+[Progress circular](https://vuetifyjs.com/en/components/progress-circular/) to
+build a count down timer.
+
+The documentation explain what elements one can change, in this case the size and
```
Comment (robmarcer) [inline src/blog/2023/10/custom-vuetify-components-dashboard.md:32]:
```suggestion
The documentation explains which elements one can change, in this case the size and
```

File: src/blog/2023/10/custom-vuetify-components-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+!["Progress spinner, random values"](./images/random-progress-element.gif "Progress spinner, random values")
+
+
+### Finishing the count down timer
+
+This is mostly a programmers job, but it's not hard, so let's get to it. A button
+would be great to reset the timer, and for the sake of this post we can hardcode
+the deadline to 1m from the button press.
+
+When draggin in a butten node, connect it to a [change](/node-red/core-nodes/change)
```
Comment (robmarcer) [inline src/blog/2023/10/custom-vuetify-components-dashboard.md:84]:
```suggestion
When dragging in a button node, connect it to a [change](/node-red/core-nodes/change)
```

**Review by robmarcer — APPROVED** [review]

**Review by robmarcer — APPROVED** [review]

**Comment by ZJvandeWeg** [comment]
@robmarcer I'll mark it as draft next time. 😄 The social tile is missing, though I bet @Yndira-FlowForge can create a PR to add that when it's done.

**Comment by Yndira-FlowForge** [comment]
Yes! That was the plan after I noticed it was merged 😉

**Comment by robmarcer** [comment]
I did check first @ZJvandeWeg. :-D

**Comment by ZJvandeWeg** [comment]
Rather this way, than delaying merging of content that's not going to make a needle moving difference. Speed of iterations I value more. So no worries!

**Pull request merged**